ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -263,17 +265,32 @@ Item {
                 closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
                 width: parent.width + Style.current.bigPadding
                 dim: false
-                Repeater {
+
+                SortFilterProxyModel {
+                    id: proxyModel
+                    sourceModel: root.startupStore.startupModuleInst.loginAccountsModel
+                    filters: ValueFilter {
+                        roleName: "keyUid"
+                        value: root.startupStore.selectedLoginAccount.keyUid
+                        inverted: true
+                    }
+                }
+
+                ListView {
```

Comment:
I would add a small comment here explaining why `ListView` is used instead of `Repeater`. It's not obvious at first glance why it's decided to use heavier ListView with height specified explicitly instead of `Repeater` :)